package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "resend",
-  "version": "4.1.1",
+  "version": "4.2.0",
```

Comment:
let's not bump the version in this pr and bump separately

Reply:
sweet! reverting it back!